USAGE.md

```diff
@@ -214,7 +218,8 @@ echo $file->getAspectRatio();
 
 #### Downloading...
 
-You can obtain the data of the file by using the `downloadData()` method and use it in your script, or write it directly to a local file via the `downloadFile()` method.
+You can obtain the data of the file by using the `downloadData()` method,
```

Comment:
A comma shouldn't be added here: either the file data from `downloadData` is used in the script, or the file data is written out via `downloadFile`. The comma suggests that after the first method, one still has a choice.

Reply:
Ah, that's a tricky construction indeed. I'll keep the "and..." part in this line to help drive home the point that this is meant to be a sequence. Maybe we can shorten the clause a bit:

> You can get the file content with the `downloadData()` method and use it in your script,

WDYT?

Reply:
Then the second part implicitly becomes: "write the file content to a local file via"... which is weird. That's why I used "the data of the file" phrase. I'd just stick with the original two-line sentence.

Reply:
Good point. [Done](https://github.com/hamstar/Wikimate/compare/e7f2f232a9c243b8ac3a7c6017562e80e165b511..656e027da9b033080894cc7bc0c4d391cc598389).